velox/CMakeLists.txt

```diff
@@ -64,3 +64,7 @@ endif()
 if(${VELOX_CODEGEN_SUPPORT})
   add_subdirectory(experimental/codegen)
 endif()
+
+if(${VELOX_ENABLE_BENCHMARKS_LARGE})
+  add_subdirectory(benchmarks_large)
```

Comment:
I'm expecting more benchmarks and wonder if we could put all of them under benchmarks folder. TPC-H could go under benchmarks/tpch.

Reply:
Would it make sense to move current code in benchmarks into benchmarks/basic ? I feel that having too many top-level directories makes it harder to browse the code base.